snuba/clickhouse.py

```diff
@@ -600,20 +599,21 @@ def get_distributed_engine(cluster, database, local_table,
     }
 
 
-def get_local_table_definition():
+def get_local_table_definition(dataset):
+    table = dataset.SCHEMA.get_local_table_name()
     return get_table_definition(
-        settings.DEFAULT_LOCAL_TABLE, get_replicated_engine(name=settings.DEFAULT_LOCAL_TABLE)
+        table, get_replicated_engine(name=table)
     )
 
 
-def get_dist_table_definition():
+def get_dist_table_definition(dataset):
     assert settings.CLICKHOUSE_CLUSTER, "CLICKHOUSE_CLUSTER is not set."
 
     return get_table_definition(
-        settings.DEFAULT_DIST_TABLE,
+        dataset.SCHEMA.get_table_name(),
```

Comment:
I just realized this can end up being the local table as well in local mode, not sure if that's worth dealing with at the moment

Reply:
this naming changes in the PR on top of this one anyway.